src/main/scala/no/ndla/draftapi/controller/DraftController.scala

```diff
@@ -406,5 +409,34 @@ trait DraftController {
       converterService.stateTransitionsToApi(user.getUser)
     }
 
+    post(
+      "/upload-file/",
```

Comment:
Foreslår å kalle denne for bare `"/file/"`. Hvis vi (mot formodning) må ha muligheten til å f.eks liste ut filer via et nytt endepunkt er det ryddigere at det ligger samlet under en felles `"/file/"` -path i stedet for et nytt et med navn `"/list-files/"`.
Ville nok også laget en ny kontroller for dette

Reply:
Det er en god idé :)

Reply:
Droppa å putte det i en egen kontroller-klasse? :)